src/web/mjs/connectors/ManHuaFen.mjs

```diff
@@ -0,0 +1,16 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class ManHuaBei extends SinMH {
```

Comment:
Class already exists, rename to `ManHuaFen`

Reply:
I have renamed it. Sorry for this small error.